tests/main/interfaces-alsa/task.yaml

```diff
@@ -0,0 +1,94 @@
+summary: Ensure that the alsa interface works.
+
+systems: [-debian-*]
```

Comment:
The test failed on the first run https://travis-ci.org/snapcore/snapd/builds/237202197#L1190 this check is with the plug disconnected https://github.com/snapcore/snapd/pull/3408/files#diff-38deae1fd3338b606e72783f027800a2R63, seems like the confinement is not working?